src/app/components/overview-history-summary-header/overview-history-summary-header.component.html

```diff
@@ -79,17 +79,15 @@ <h2 class="sr-only">Git stats</h2>
                     <li class="ml-0 ml-md-3">
                         <a
                             data-pjax="#repo-content-pjax-container"
-                            href="#todo-navigate-to-history"
+                            (click)="navigateToHistory(metadataBlockFragment.author.accountName, this.datasetName)"
                             class="pl-3 pr-3 py-3 p-md-0 mt-n3 mb-n3 mr-n3 m-md-0 link--primary text-decoration-none flex-nowrap"
                         >
                             <span class="mr-1">
                                 <svg-icon name="history"></svg-icon>
                             </span>
                             <span class="d-none d-sm-inline">
                                 <strong>{{ numBlocksTotal }}</strong>
-                                <span aria-label="Commits on KAMU-58" class="color-fg-muted d-none d-lg-inline">
-                                    blocks
-                                </span>
+                                <span aria-label="Commits on KAMU-58"> blocks </span>
```

Comment:
What a hell is this?

Reply:
I modified small bug with hover effect and added redirect to History tab

![image](https://github.com/kamu-data/kamu-web-ui/assets/51016717/bd827d8b-06dc-49b7-99e9-2b0e38362a46)